### PR TITLE
Failed to load application files message improvement

### LINKED ISSUE #26128: Improve "failed to load" message when serving at subpath
If grafana fails with subpath, it would be nice to show a hint about `serve_from_sub_path`.

**What happened**:

When serving grafana at subpath (from kube-prometheus with ingress), the following message is shown in the browser. There are no 404s or anything like that:

```
If you're seeing this Grafana has failed to load its application files

[...]

2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath

[...]
```

**What you expected to happen**:

A message like the following would save a lot of time in fixing the issue:

```
If you're seeing this Grafana has failed to load its application files

[...]

2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath,
and that serve_from_sub_path is set unless there is a rewriting proxy in front of grafana.

[...]
```

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #26128

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by dprokop** [comment]
@papagian is this going to be included in 7.1?

**Comment by papagian** [comment]
> @papagian is this going to be included in 7.1?

not sure, I've assigned it to 7.2 since it's just an improvement.

File: public/views/index-template.html [hunk 1]
```diff
@@ -199,9 +199,10 @@
         </p>
         <p>
           1. This could be caused by your reverse proxy settings.<br /><br />
-          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath<br />
+          2. Otherwise, if you host grafana under subpath make sure your grafana.ini root_url setting includes subpath
```
Comment (marefr) [inline public/views/index-template.html:202]: Otherwise to what? Using a reverse proxy?

File: public/views/index-template.html [hunk 2]
```diff
@@ -199,9 +199,10 @@
...
+          and that serve_from_sub_path is set.<br />
```
Comment (marefr) [inline public/views/index-template.html:203]: serve_from_sub_path shouldn't be needed when using a reverse proxy, but could. You use serve_from_sub_path if you want to host under a sub path but don't use a reverse proxy.

**Review by marefr — CHANGES_REQUESTED** [review]
Some minor comments/concerns

File: public/views/index-template.html (hunk 2 above)
Comment (papagian) [reply public/views/index-template.html:203]: that's I have used `otherwise`. Should I change it to: `I you don't use a reverse proxy...`?

Comment (marefr) [reply public/views/index-template.html:203]: I'm confused by the whole issue request. If you're not using a reverse proxy why would you end up with this problem?

Comment (papagian) [reply public/views/index-template.html:203]: I guess the [issue](https://github.com/grafana/grafana/issues/26128) is for adding an indication for making sure that `serve_from_sub_path` is set in addition to setting `root_url` properly  if not using a reverse proxy, no?

File: public/views/index-template.html (hunk 1 above)
Comment (marefr) [reply public/views/index-template.html:202]: @papagian what about this?

```suggestion
          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not using a reverse proxy make sure to set serve_from_sub_path to true.
```

File: public/views/index-template.html (hunk 2 above)
Comment (papagian) [inline public/views/index-template.html:203]:
```suggestion
```

File: public/views/index-template.html [hunk 3]
```diff
@@ -199,7 +199,7 @@
         </p>
         <p>
           1. This could be caused by your reverse proxy settings.<br /><br />
-          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath<br />
+          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not using a reverse proxy make sure to set serve_from_sub_path to true.
```
Comment (aknuds1) [inline public/views/index-template.html:202]: Is there a reason why the `<br />` at the end of the line is removed? Seems that other list items end with to `<br />`.

Comment (papagian) [reply public/views/index-template.html]: good catch; it was by mistake

**Review by aknuds1 — APPROVED** [review]
LGTM

**Review by hugohaggmark — APPROVED** [review]
LGTM!

**Comment by papagian** [comment]
@marefr Can you have a fresh look next week? thanks!

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**